cmake/compile_definitions/linux.cmake

```diff
@@ -189,12 +189,29 @@ if(X11_FOUND)
             "${CMAKE_SOURCE_DIR}/src/platform/linux/x11grab.cpp")
 endif()
 
+# XDG portal
+if (${SUNSHINE_ENABLE_PORTAL})
+    pkg_check_modules(GIO gio-2.0 gio-unix-2.0)
+    pkg_check_modules(PIPEWIRE libpipewire-0.3)
```

Comment:
Runtime deps will need to be added here: https://github.com/LizardByte/Sunshine/blob/master/cmake/packaging/linux.cmake#L43-L73

And for packaging need to be added in these different places:

- https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/scripts/linux_build.sh#L88
- https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/scripts/linux_build.sh#L151
- https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/packaging/sunshine.rb#L40
- https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/packaging/linux/Arch/PKGBUILD#L15
- https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/packaging/linux/fedora/Sunshine.spec#L18
- https://github.com/LizardByte/Sunshine/tree/master/packaging/linux/flatpak/modules (probably) and https://github.com/LizardByte/Sunshine/blob/7dd836dab63e15db54f18ed2b64cb394aa30c308/packaging/linux/flatpak/dev.lizardbyte.app.Sunshine.yml#L31